File: c/lanche.c
#include <stdio.h>
int main()
{
	float custos = 0;
    int a;
    scanf("%d",&a);
    switch(a){
        case 1:
            custos += 4;
            break;
        case 2:
            custos += 4.5;
            break;
        case 3:
            custos += 5;
            break;
        case 4:
            custos += 2;
            break;
        case 5:
            custos += 1.5;
            break;        
    }

    scanf("%d",&a);
    custos *= a;
    
    printf("Total: R$ %.2f\n",custos);

	return 0;
}

File: python/lanche.py
dic = {
    1 : 4,
    2 : 4.5,
    3 : 5,
    4 : 2,
    5 : 1.5,
}
custos = 0 
a = [int(x) for x in input().split()]
custos += dic[a[0]]
custos *= a[1]
print("Total: R$ {:0.2f}".format(custos))